__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const {
+  categoryData,
+  reviewData,
+  userData,
+  commentData,
+} = require("../db/data/test-data/index.js");
+const request = require("supertest");
+const { app, server } = require("../app.js");
```

Comment:
You shouldn't need to import both an `app` and an `server` here - essentially, the app _is_ the server.
Can see you are invoking the `listen` method in app.js - you don't need to do this either as `supertest` will start your app listening for you.
I would advise that you extract the listening behaviour into a separate file so that when the app is required, the listening doesn't begin